docs/conf.py

```diff
@@ -41,6 +41,7 @@
 import searx.plugins
 import searx.webutils
 searx.engines.load_engines(searx.settings['engines'])
+searx.plugins.initialize(None)
```

Comment:
`app=None` would be more readable, otherwise LGTM

Reply:
Even if it is not needed by the plugins we have right now, I decided to import the Flask app from webapp.py / @not-my-profile what do you think (613fb155)?